manifests/gemrc.pp

```diff
@@ -17,7 +17,7 @@
       ensure => directory,
       owner  => 'root',
       group  => 'root',
-      mode   => '0644',
+      mode   => '0755',
```

Comment:
side note, these two lines are exactly equivalent, due to permission promotion. See the last bullet of https://docs.puppet.com/puppet/latest/types/file.html#file-attribute-mode

> When specifying numeric permissions for directories, Puppet sets the search permission wherever the read permission is set.

This means that `'0644'` gets promoted to `'0755'`. Not that it matters either way in the code here :)

Reply:
Yeah, I thought of that, then I figured to still put in the `0755`. Pedantic++ 😆